C:\Users\UMI\Desktop\disrl\myenv\Lib\site-packages\gym\utils\passive_env_checker.py:233: DeprecationWarning: `np.bool8` is a deprecated alias for `np.bool_`.  (Deprecated NumPy 1.24)
  if not isinstance(terminated, (bool, np.bool8)):
Traceback (most recent call last):
  File "C:\Users\UMI\Desktop\disrl\qr-dqn.py", line 173, in <module>
    loss = train_step(batch)  # Define loss here
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\UMI\Desktop\disrl\qr-dqn.py", line 73, in train_step
    loss = quantile_huber_loss(target_quantiles, action_quantiles, delta=model.delta)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UMI\Desktop\disrl\qr-dqn.py", line 46, in quantile_huber_loss
    y_true = tf.reshape(y_true, tf.shape(y_pred))  # Reshape if necessary
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UMI\Desktop\disrl\myenv\Lib\site-packages\tensorflow\python\ops\weak_tensor_ops.py", line 88, in wrapper
    return op(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UMI\Desktop\disrl\myenv\Lib\site-packages\tensorflow\python\util\traceback_utils.py", line 153, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\UMI\Desktop\disrl\myenv\Lib\site-packages\tensorflow\python\framework\ops.py", line 5983, in raise_from_not_ok_status
    raise core._status_to_exception(e) from None  # pylint: disable=protected-access
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tensorflow.python.framework.errors_impl.InvalidArgumentError: {{function_node __wrapped__Reshape_device_/job:localhost/replica:0/task:0/device:CPU:0}} Input to reshape is a tensor with 32 values, but the requested shape has 6400 [Op:Reshape] name:
Episode 1: Reward: 16.0, Loss: None